src/plugins/intel_gna/src/layers/gna_split_layer.hpp

```diff
@@ -73,22 +71,21 @@ inline std::pair<int64_t, std::vector<uint32_t>> AlignedSplitSizesPerAxis(Infere
     IE_ASSERT(firstValuableDim != std::end(dims));
     auto splittedElementsSize = *firstValuableDim;
     auto splittedDimIx = std::distance(std::begin(dims), firstValuableDim);
-    auto alignment = limitations::Limitations::kInputByteAlignment;
+    auto alignment = limitations::Limitations::get_instance()->get_memory_alignment();
 
-    // Split output size should be multiple by 64 to avoid align filters insertion,
-    // but we need to check if our input size to split exceeds 64; if not we can always
+    // Split output size should be multiple of device memory alignment to avoid align filters insertion,
+    // but we need to check if our input size to split exceeds alignment; if not we can always
     // split if the remaining size is aligned
-    if (splittedElementsSize <= alignment) {
+    auto max_split_size = limitations::Limitations::kBufferMaxSize * splittedElementsSize / totalElementsSize;
```

Comment:
It would be nice to add some explanation to the variable names or make the names more self-explanatory.

Reply:
Okay, I will change the variable name to "split_size" and modify the description of the "GetAlignedSplitSizes" function